js/vote.js

```diff
@@ -55,9 +55,20 @@ function switchSidebar() {
 }
 
 $(document).ready(function () {
-	// count how many times in each date
 	new Clipboard('.copy-link');
+	// count how many times in each date
 	updateBest();
+
+	// Temporary hack - Check if we have Nextcloud or ownCloud with an anomymous user
+	var $hideAvatars = false;
+	if (!document.getElementById('nextcloud')) {
+		$product = 'ownCloud';
```

Comment:
$ indicates that the variable holds a jQuery object and this variable seems not to be used anywhere. And we should use single quotes in JS and PHP where possible (next line). Besides from that nice hack :D.